src/flowcean/river/river_learner.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import pickle
+
+import polars as pl
+from river.base import Regressor
+from typing_extensions import override
+
+from flowcean.core import Model, SupervisedLearner
+
+# from flowcean.polars import to_numpy
+
+logger = logging.getLogger(__name__)
+
+
+class RiverModel(Model):
+    def __init__(self, model: Regressor, output_column: str) -> None:
+        self.model = model
+        self.output_column = output_column
+
+    @override
+    def predict(self, inputs: pl.DataFrame) -> pl.DataFrame:
```

Comment:
this should probably be a `LazyFrame`
```suggestion
    def predict(self, inputs: pl.LazyFrame) -> pl.LazyFrame:
```

or both, as you are collecting them afterwards